xil_cloud/docker-compose.yml

```diff
@@ -1,9 +1,8 @@
-version: '3'
+version: '2'
 services:
-
   carma-simulation:
     privileged: true
-    image: usdotfhwastol/carma-simulation:carma-simulation-1.0.1
+    image: usdotfhwastoldev/carma-simulation:develop
     networks:
```

Comment:
Not pretty sure but is that needed to add `carma_net_1` here since carma-simulation is going to communicate with carma-platform

Reply:
Fixed this by putting the ns3_adapter on both the carma-net and carma-sim-net